tests/writer_util.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at:
+#
+#    http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+# Python 2/3 compatibility
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from datetime import datetime
+from decimal import Decimal
+import six
+from pytest import raises
+
+from amazon.ion.core import IonEventType, IonEvent
+from amazon.ion.core import IonType
+from amazon.ion.util import record
+from amazon.ion.writer import WriteEventType
+from tests import is_exception
+from tests import noop_manager
+
+
+_D = Decimal
+_DT = datetime
+
+_E = IonEvent
+_IT = IonType
+_ET = IonEventType
```

Comment:
Any names you expect to have a user use should not have a `_` prefix, I am not sure which names fit the criteria.